lms/djangoapps/certificates/models.py

```diff
@@ -911,34 +941,33 @@ def set_self_generatation_enabled_for_course(cls, course_key, is_enabled):
         )
 
     @classmethod
-    def is_language_specific_templates_enabled_for_course(cls, course_key):
```

Comment:
Note: I don't think there's anything wrong with having these static methods, just that we should be careful about how we use them. If we only need to access one field, they're fine. If we need access to more than one field to satisfy a request, it's probably better to use something else so that we don't have to hit the DB with extra queries.

Reply:
Right ok, that make sense. So I should add that methods back to the model?

Reply:
No, you don't have to if they're not being used. I just wanted to point out that it's ok to use them in certain situations.